libsolidity/ast/AST.cpp

```diff
@@ -215,14 +215,14 @@ vector<Declaration const*> const& ContractDefinition::inheritableMembers() const
 {
 	if (!m_inheritableMembers)
 	{
-		set<string> memberSeen;
+		set<size_t> memberSeen;
 		m_inheritableMembers.reset(new vector<Declaration const*>());
 		auto addInheritableMember = [&](Declaration const* _decl)
 		{
 			solAssert(_decl, "addInheritableMember got a nullpointer.");
-			if (memberSeen.count(_decl->name()) == 0 && _decl->isVisibleInDerivedContracts())
```

Comment:
I actually don't think the first check is even necessary now.

Reply:
True, the entire map can be removed.